cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -168,6 +168,16 @@ class LeadImpl extends ServerImpl with Lead
             throw e;
         }
       }
+
+      // The auth service is not yet initialized at this point.
+      // So simply check the auth-provider property value.
+      if (checkAuthProviderProp(bootProperties)) {
+        logDebug(" LDAP Security is Enabled :: Setting authenticator for SnappyData Pulse")
```

Comment:
Can put info message saying something like "Enabling authentication for Spark UI (or SnappyData Pulse) users." and skip it for else case.